qcore/timeseries.py

```diff
@@ -293,12 +293,16 @@ def __init__(self, outbin):
         outbin: path to OutBin folder containing seis files
         """
         self.seis = sorted(glob(os.path.join(outbin, "*seis-*.e3d")))
+        #try load e3d.par at the same directory first
+        #otherwise try look for one folder above
         self.e3dpar = os.path.join(outbin, "e3d.par")
         if not os.path.isfile(self.e3dpar):
-            raise ValueError(
-                "The given directory does not contain an e3d.par file. "
-                "Either move or create a symlink to the correct file please."
-            )
+            self.e3dpar = os.path.join(outbin, "../e3d.par")
```

Comment:
In an ideal world I don't think we should be looking outside the given directory, but this is ok as a "It's gets the job done" fix I think. Maybe add a flag to the user that it had to look outside?